BlueprintUICommonControls/Sources/Label.swift

```diff
@@ -12,18 +12,38 @@ public struct Label: Element {
     public var alignment: NSTextAlignment = .left
     public var numberOfLines: Int = 0
     public var lineBreakMode: NSLineBreakMode = .byWordWrapping
-    /// The scale to which pixel measurements will be rounded. Defaults to `UIScreen.main.scale`.
-    public var roundingScale: CGFloat = UIScreen.main.scale
+    public var lineHeight: LineHeight = .font
+    public var isAccessibilityElement = false
 
     public init(text: String, configure: (inout Label) -> Void = { _ in }) {
         self.text = text
         configure(&self)
     }
+
+    private var effectiveLineBreakMode: NSLineBreakMode {
+        // These line break modes don't work when numberOfLines is 1, and they break line height adjustments.
+        // Normalize them to clipping mode instead.
+        if numberOfLines == 1 && (lineBreakMode == .byCharWrapping || lineBreakMode == .byWordWrapping) {
```

Comment:
I discovered that the text rect offset we're using to support line height is ignored by UILabel in these specific cases. UILabel does not "wrap" these cases at all, but treats them like clipping mode, so I think we can work around this by normalizing to clipping mode.

I've updated the snapshot tests for custom line heights to validate it works in various configurations.